auto/configure.py

```diff
@@ -196,6 +196,10 @@ def generate_user_auth(self, password):
         with open(self._config_path / 'user.key', "wb") as f:
             f.write(pem)
 
+        user_public_key = """---\r\npubkey: {0}""".format(public_key.decode('utf-8'))
```

Comment:
Skip the prefix, that gets added on import.
We need to import it as a valid rsa key.

Reply:
Done